mango-viz/bdgenomics/mango/io/bedfile.py

```diff
@@ -47,4 +46,7 @@ def _to_json(cls, df):
         #ending json
         json_ga4gh = json_ga4gh + "]}"
         return json_ga4gh
-        #raise NotImplementedError("Implement Me!")
+    
+    @classmethod
+    def _visualization(cls, df):
+        return 'featureJson'

```

Comment:
We have been talking about adding unit tests for a while, can you add some to this PR?